app/controllers/groups_controller.rb

```diff
@@ -4,6 +4,14 @@ class GroupsController < ApplicationController
   def index
     @groups = Group.all
     @neighborhoods = Neighborhood.all
+    my = params[:format]
+    if my.nil?
+      @all = "selected = \"selected\"".html_safe
```

Comment:
@HacksForHugs if these two cases are doing the same thing, which they seem to be doing, do you need conditions for them? why not just declare the two variables?